Update views.py

leong96 opened the pull request:
Switch index to search/search.html.

leong96 committed:
Update views.py

Switch index to search/search.html

justwheel commented on grasa_event_locator/views.py:
```diff
@@ -187,7 +187,7 @@ def event(request, eventID):
 def login(request):
         if request.user.is_authenticated:
-                return HttpResponseRedirect("index.php")
+                return HttpResponseRedirect("search/search.html")
```
This change makes sense to me, but we should use the `reverse()` method to dynamically set the URL to the internal page name, so if the URI ever changes in the future, Django will automatically handle that. Get a deeper explanation of what I mean in [this blog post](https://overiq.com/django-1-10/redirecting-urls-in-django/):

`Instead of manually typing the URLs, we can also use reverse() method with HttpResponseRedirect to automatically create urls.`

This needs to be tested, but I think the fix would be `reverse(haystack_search)` for the parameter here, based on the `urls.py` file:

https://github.com/jwflory/django-rit-grasa/blob/c0817411f626378fb77e7cbcf381aaf1c7a6a89e/grasa_event_locator/urls.py#L37

justwheel reviewed (changes requested):
Hey @leong96, this looks good! I had one specific suggestion with a way to make it less likely for a URL to break accidentally. Let me know what you think.

leong96 committed:
Merge branch 'master' into hleong_index_fix

leong96 commented:
@jwflory That's good to know, gives the name of the URL an actual purpose now. I've switched to the reverse method, and it tests fine.

leong96 committed:
Update views.py

Switched from writing the URL redirecting to the search to using the reverse method, which instead requires the name specified in the url file.

leong96 committed:
Merge branch 'hleong_index_fix' of https://github.com/jwflory/django-rit-grasa into hleong_index_fix

justwheel reviewed (approved):
Cool, looks good to me! Thanks for getting this one so fast. Merging! :ocean:
